### PR TITLE
Introduce faster app CRs status change to the next Azure release

### PR DESCRIPTION
**Pull request by tomahawk28**
To bring giantswarm/roadmap#191 into TCs.

Related slack announcement (when we deploy this to CPs) : https://gigantic.slack.com/archives/C04TGHDEF/p1604052483005300

### CODE REVIEW

File: azure/requests.yaml [hunk 1]
```diff
@@ -1,4 +1,12 @@
 releases:
+    - name: "> 13.0.0-beta1"
```
Comment (tomahawk28) [inline azure/requests.yaml:2]: @giantswarm/team-celestial Is this okay to set the beta version as a threshold?

**Review by whites11 — APPROVED** [review]

**Review by cokiengchiara — APPROVED** [review]
LGTM since @whites11 is ok with 
`    - name: "> 13.0.0-beta1"`

File: azure/requests.yaml (hunk 1 above)
Comment (whites11) [reply azure/requests.yaml:2]: I don't see why not

Comment (tomahawk28) [reply azure/requests.yaml:2]: Okay, let's go with this!

**Pull request merged**